mmdet/models/anchor_heads/anchor_head.py

```diff
@@ -152,6 +166,177 @@ def get_anchors(self, featmap_sizes, img_metas, device='cuda'):
 
         return anchor_list, valid_flag_list
 
+    def anchor_target_single(self,
+                             flat_anchors,
+                             valid_flags,
+                             gt_bboxes,
+                             gt_bboxes_ignore,
+                             gt_labels,
+                             img_meta,
+                             label_channels=1,
+                             unmap_outputs=True):
+        """Compute regression and classification targets for anchors in
+            a single image.
+
+        Args:
+            flat_anchors (Tensor): Multi level anchors of the image,
+                shape (num_anchors ,4).
```

Comment:
Multi-level anchors of the image, which are concatenated into a single tensor of shape (num_anchors ,4).